README.md

```diff
@@ -47,6 +47,27 @@ pandoc -s --toc \
     -o README.md
 ```
 
+## [Validator API specification](specs/validator/)
+
+### Building
+
+The OpenAPI specification document is used to generate clients and services that drive the Tabeland Validator. It can also be used to generate Markdown documentation via:
+
+```shell
+npm i -g widdershins
+cd specs/validator
+widdershins tableland-openapi-spec.yaml \
+    --omitHeader \
+    --httpsnippet \
+    --language_tabs \
+    'javascript:JavaScript:fetch' 'javascript--node:Node.JS' 'go:Go' 'shell:cURL:curl' \
```

Comment:
This is just a comment, but maybe it's good only to include `shell:cURL;curl` since we have nicer SDKs for JS and Go?

Reply:
yeh good call. it is also kinda messy with that many options.